include/sentry.h

```diff
@@ -1346,6 +1346,45 @@ SENTRY_EXPERIMENTAL_API sentry_uuid_t sentry_transaction_finish(
  * into this function.
  */
 SENTRY_EXPERIMENTAL_API void sentry_set_span(sentry_value_t transaction);
+
+/**
+ * Starts a new Span.
+ *
+ * Either the return value of `sentry_start_transaction`, `sentry_set_span`, or
```

Comment:
```suggestion
 * Either the return value of `sentry_start_transaction`, or
```

That fn was changed recently